src/app-layer-smtp.c

```diff
@@ -147,7 +147,6 @@ SCEnumCharMap smtp_decoder_event_table[] = {
 
     /* MIME Events */
     { "MIME_PARSE_FAILED", SMTP_DECODER_EVENT_MIME_PARSE_FAILED },
-    { "MIME_MALFORMED_MSG", SMTP_DECODER_EVENT_MIME_MALFORMED_MSG },
```

Comment:
need a graceful way of failing event rules for this event, did you test that? SV?

Reply:
> need a graceful way of failing event rules for this event,

Sorry I do not understand this sentence.

Do you mean that there should be a warning that the rule `#alert smtp any any -> any any (msg:"SURICATA SMTP Mime malformed message found"; flow:established; app-layer-event:smtp.mime_malformed_msg; flowint:smtp.anomaly.count,+,1; classtype:protocol-command-decode; sid:2220010; rev:1;)` will never trigger ?